cleverpush/src/main/java/com/cleverpush/banner/AppBannerModule.java

```diff
@@ -84,6 +84,7 @@ public class AppBannerModule {
   private String pendingShowAppBannerNotificationId = null;
   private Set<String> bannerClickedList = new HashSet<>();
   private Set<String> bannerDeliveredList = new HashSet<>();
+  public String currentEventId = null;
```

Comment:
Not getting why we need a global variable for this, this seems wrong to me.

Can't we just remove the condition here: https://github.com/cleverpush/cleverpush-android-sdk/blob/master/cleverpush/src/main/java/com/cleverpush/banner/AppBannerModule.java#L731

Reply:
> Not getting why we need a global variable for this, this seems wrong to me.
> 
> Can't we just remove the condition here: https://github.com/cleverpush/cleverpush-android-sdk/blob/master/cleverpush/src/main/java/com/cleverpush/banner/AppBannerModule.java#L731

@unnaticleverpush probably not that simple, but we can check together next time with Milad if you want